### PR TITLE
Prism instead of Highlight

### PR DESCRIPTION
**Pull request by lipis**
http://prismjs.com/

```
highlight: wire-vendor.min.js: 2,192,578 bytes 
prism:     wire-vendor.min.js: 1,702,730 bytes 
```

makred.js needs this patch though..

```js
  marked.defaults = {
    ...
    highlight: function (code) {
      return Prism.highlight(code, Prism.languages.javascript);
    },
    ...
  };
```

### CODE REVIEW

File: app/style/content/conversation/message-list.less [hunk 1]
```diff
@@ -480,3 +480,7 @@
   font-family: @font-family-ephemeral;
   color: @ephemeral-color;
 }
+
+code {
+  font-size: .9em;
+}
```
Comment (lipis) [inline app/style/content/conversation/message-list.less:486]: not sure about this one.. we could ignore it :)

**Comment by herrmannplatz** [comment]
3

**Comment by lipis** [comment]
2

**Comment by herrmannplatz** [comment]
1

**Pull request closed without merging**